<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SENanotubeCreatorEditorGUIClass</class>
 <widget class="QWidget" name="SENanotubeCreatorEditorGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>206</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>177</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>460</width>
    <height>212</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Length</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Number of Double Strand DNA</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" rowspan="2" colspan="2">
      <widget class="QComboBox" name="comboBoxRouting">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Seamless</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tiles</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Non-seamless</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0" rowspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Routing</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxRadius">
       <property name="suffix">
        <string> nm</string>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="spinBoxNumberOfDSDNA">
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Radius</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="spinBoxNumberOfBasePairs">
       <property name="suffix">
        <string> bp</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999999</number>
       </property>
       <property name="value">
        <number>21</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="3">
      <widget class="QCheckBox" name="chkPredefined">
       <property name="text">
        <string>Predefined size</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="QTextEdit" name="txtInfo">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>chkPredefined</sender>
   <signal>toggled(bool)</signal>
   <receiver>SENanotubeCreatorEditorGUIClass</receiver>
   <slot>onPredefinedNanotube(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxNumberOfBasePairs</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SENanotubeCreatorEditorGUIClass</receiver>
   <slot>onNumberOfBasePairsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBoxRadius</sender>
   <signal>valueChanged(double)</signal>
   <receiver>SENanotubeCreatorEditorGUIClass</receiver>
   <slot>onRadiusChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxNumberOfDSDNA</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SENanotubeCreatorEditorGUIClass</receiver>
   <slot>onNumberOfDSDNAChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxRouting</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SENanotubeCreatorEditorGUIClass</receiver>
   <slot>onChangeRouting()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onChangeRouting()</slot>
  <slot>onPredefinedNanotube(bool)</slot>
  <slot>onRadiusChanged()</slot>
  <slot>onNumberOfBasePairsChanged()</slot>
  <slot>onNumberOfDSDNAChanged()</slot>
 </slots>
</ui>
